app/controllers/schemes_controller.rb

```diff
@@ -9,7 +9,7 @@ class SchemesController < ApplicationController
 
   def index
     redirect_to schemes_organisation_path(current_user.organisation) unless current_user.support?
-    all_schemes = Scheme.order(confirmed: :asc, service_name: :asc)
+    all_schemes = Scheme.order("confirmed ASC NULLS FIRST", service_name: :asc)
```

Comment:
```suggestion
    all_schemes = Scheme.order("confirmed ASC NULLS FIRST", service_name: :asc)
```

Why do we have `nil` confirmed values? It shouldn't be possible.

Reply:
Hm, we only set confirmed to true if it's confirmed, otherwise we don't set it at all 🤷‍♀️

Reply:
I think what you had previously would work. What do you think?

Reply:
The tests would fail if I removed this. I think by default when ordering the nulls with ASC they go in the back `SQL considers NULL to be a value less than any present value`

Reply:
Which after reading this sentence it doesn't make sense (but that's how it works)